### PR TITLE
Fix pushing to 1-x-dev docker tag

### PR DESCRIPTION
**Pull request by zeripath**
It appears that #18551 and #18573 have a mistake in that raymond does not have
an {{else}} on {{#equal}}. This PR notes that Sprig has a hasPrefix function
and so we use this with another if.

Signed-off-by: Andrew Thornton <art27@cantab.net>

### CODE REVIEW

**Comment by silverwind** [comment]
Are these supposed to be actual Handlebars files? I would suggest renaming to `.hbs` if possible to not confuse them with go templates.

**Comment by zeripath** [comment]
It appears that they are handlebars formatted files.

Drone runs the drone-manifest plugin which passes the files through https://github.com/aymerick/raymond.

Now I'm loathe to make more changes to drone until this is actually correct.

**Comment by silverwind** [comment]
I see, so the issues is that even the [example](https://plugins.drone.io/drone-plugins/drone-manifest/) incorrectly uses `.tmpl` (suggesting go template) for what is actually Handlebars. Not a big issue, feel free to ignore.

**Review by silverwind — APPROVED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -826,7 +826,7 @@ steps:
       tags: dev-linux-amd64
       repo: gitea/gitea
       build_args:
-        - GOPROXY=https://goproxy.cn
+        - GOPROXY=off
```
Comment (zeripath) [inline .drone.yml]:
```suggestion
        - GOPROXY=https://goproxy.cn
```

File: .drone.yml [hunk 2]
```diff
@@ -843,6 +843,70 @@ steps:
       auto_tag: false
       tags: dev-linux-amd64-rootless
       repo: gitea/gitea
+      build_args:
+        - GOPROXY=off
```
Comment (zeripath) [inline .drone.yml]:
```suggestion
        - GOPROXY=https://goproxy.cn
```

File: .drone.yml [hunk 3]
```diff
@@ -843,6 +843,70 @@ steps:
...
+      password:
+        from_secret: docker_password
+      username:
+        from_secret: docker_username
+    environment:
+      PLUGIN_MIRROR:
+        from_secret: plugin_mirror
```
Comment (zeripath) [inline .drone.yml]:
```suggestion
```

File: .drone.yml [hunk 4]
```diff
@@ -983,11 +1047,14 @@ steps:
       tags: dev-linux-arm64
       repo: gitea/gitea
       build_args:
-        - GOPROXY=https://goproxy.cn
+        - GOPROXY=off
```
Comment (zeripath) [inline .drone.yml]:
```suggestion
        - GOPROXY=https://goproxy.cn
```

File: .drone.yml [hunk 5]
```diff
@@ -983,11 +1047,14 @@ steps:
...
       password:
         from_secret: docker_password
       username:
         from_secret: docker_username
+    environment:
+      PLUGIN_MIRROR:
+        from_secret: plugin_mirror
```
Comment (zeripath) [inline .drone.yml]:
```suggestion
```

File: .drone.yml [hunk 6]
```diff
@@ -1000,6 +1067,52 @@ steps:
       auto_tag: false
       tags: dev-linux-arm64-rootless
       repo: gitea/gitea
+      build_args:
+        - GOPROXY=off
```
Comment (zeripath) [inline .drone.yml]:
```suggestion
        - GOPROXY=https://goproxy.cn
```

File: .drone.yml [hunk 7]
```diff
@@ -1000,6 +1067,52 @@ steps:
...
+      password:
+        from_secret: docker_password
+      username:
+        from_secret: docker_username
+    environment:
+      PLUGIN_MIRROR:
+        from_secret: plugin_mirror
```
Comment (zeripath) [inline .drone.yml]:
```suggestion
```

**Review by 6543 — APPROVED** [review]

**Pull request merged**